python/Ganga/Core/InternalServices/Coordinator.py

```diff
@@ -52,9 +52,10 @@ def isCredentialRequired(credObj):
         return Workspace_runtime.requiresAfsToken() or Repository_runtime.requiresAfsToken()
 
     if getName(credObj) == 'GridProxy':
-        from Ganga.GPI import jobs, typename
+        from Ganga.Core.GangaRepository import getRegistryProxy
+        from Ganga.Runtime.GPIFunctions import typename
         from Ganga.GPIDev.Base.Proxy import stripProxy
-        for j in jobs:
+        for j in getRegistryProxy('jobs'):
```

Comment:
There is different behavior. Besides the code was previously expecting a proxy wrapped object so it would be wrong to replace it without being careful.
I want to remove the raw calls to the GPI here and delay asking if it was even correct until a future PR fixing these issues.